src/modelmanager.cpp

```diff
@@ -223,6 +223,43 @@ void ModelManager::startCleaner() {
 Status ModelManager::startFromConfig() {
     auto& config = ovms::Config::instance();
 
+    this->setRootDirectoryPath("");
+    Status status = StatusCode::OK;
+
+#if (MEDIAPIPE_DISABLE == 0)
+    // Check if config is present for mediapipe graph
+    MediapipeGraphConfig mpConfig;
+    mpConfig.setGraphName(config.modelName());
+    if (config.modelPath().back() == FileSystem::getOsSeparator().back()) {
+        mpConfig.setBasePath(config.modelPath());
+        mpConfig.setRootDirectoryPath(config.modelPath());
+    } else {
+        mpConfig.setBasePath(config.modelPath() + FileSystem::getOsSeparator());
+        mpConfig.setRootDirectoryPath(config.modelPath() + FileSystem::getOsSeparator());
+    }
+    mpConfig.setGraphPath(mpConfig.getBasePath() + DEFAULT_GRAPH_FILENAME);
+    std::vector<MediapipeGraphConfig> mediapipesInConfigFile;
+
+    std::ifstream ifs(mpConfig.getGraphPath());
+    if (ifs.is_open()) {
+        mpConfig.setSubconfigPath(DEFAULT_SUBCONFIG_FILENAME);
+        SPDLOG_LOGGER_DEBUG(modelmanager_logger, "Adding mediapipe graph config for {}, {}", mpConfig.getGraphName(), mpConfig.getGraphPath());
+        mediapipesInConfigFile.push_back(mpConfig);
+        std::vector<ModelConfig> gatedModelConfigs;
+        std::set<std::string> modelsInConfigFile;
+        std::set<std::string> modelsWithInvalidConfig;
+        std::unordered_map<std::string, ModelConfig> newModelConfigs;
+        loadMediapipeSubConfigModels(gatedModelConfigs, modelsInConfigFile, modelsWithInvalidConfig, newModelConfigs, mediapipesInConfigFile);
+
+        this->servedModelConfigs = std::move(newModelConfigs);
```

Comment:
Below I can see:
```
        ModelConfig{
            config.modelName(),
            config.modelPath(),
            config.targetDevice(),
            config.batchSize(),
            config.nireq(),
            config.stateful(),
            config.idleSequenceCleanup(),
            config.lowLatencyTransformation(),
            config.maxSequenceNumber(),
            this->modelCacheDirectory});
```

Should we reject such CLI options? Or ignore as we do now? @dtrawins 

Example:

`ovms --rest_port 9000 --model_name llama --model_path llama/ --batch_size 10` this will work now, right? @rasapala  but probably shouldnt

Reply:
Decided to return error if model argument was added.